### PR TITLE
[charts/portal] readme correction 

### PR DESCRIPTION
**Pull request by premchandervodela**
**Description of the change**

**Benefits**

**Drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Review by murthyravi14 — APPROVED** [review]
LGTM

**Review by kk632390 — COMMENTED** [review]
It will be good if we change the Header and following sentence to have text Ingress-nginx as the subchart is also of that name
- Nginx-Ingress (Header ##)
- The following table lists the configured parameters of the **Nginx-Ingress** Subchart - https://github.com/kubernetes/ingress-nginx

File: charts/portal/README.md [hunk 1]
```diff
@@ -503,23 +503,22 @@ The following table lists the configured parameters of the MySQL Subchart - http
 | `mysql.primary.configuration`   | MySQL Primary configuration to be injected as ConfigMap	   | `see values.yaml` |
 
 
-## Nginx-Ingress
-The following table lists the configured parameters of the Nginx-Ingress Subchart - https://github.com/helm/charts/tree/master/stable/nginx-ingress
```
Comment (APIM-KiranVaddadi) [inline charts/portal/README.md:507]: may be  https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx

**Review by APIM-KiranVaddadi — COMMENTED** [review]
Version numbers

**Comment by premchandervodela** [comment]
Have raised a new pr 114 for this change - and hence closing this.

**Pull request closed without merging**